schemas/backend_configuration_schema.json

```diff
@@ -167,141 +167,159 @@
                     "type": "integer",
                     "description": "Backend quantum volume",
                     "minimum": 1
+                    },
+                "processor_type": {
+                    "type": "object",
+                    "required": ["family", "revision", "segment"],
+                    "properties": {
+                        "family": {
+                            "type": "string",
+                            "description": "Processor family"
+                        },
+                        "revision": {
+                            "type": "number",
+                            "description": "Processor revision number"
+                        },
+                        "segment": {
+                            "type": "string",
+                            "description": "Processor family"
```

Comment:
Copy & paste error here. It'd be nice to be more descriptive than "processor segment", since I for one don't know what that means.

Reply:
Fixed in 4b9078d. Added more detail on segment.